chatkit-core/src/main/kotlin/com/pusher/chatkit/ChatkitCore.kt

```diff
@@ -0,0 +1,77 @@
+package com.pusher.chatkit
+
+import com.pusher.chatkit.api.createPlatformClientModule
+import com.pusher.chatkit.state.createStoreModule
+import com.pusher.chatkit.users.api.createUserSubscriberModule
+import com.pusher.util.Result
+import com.pusher.util.asSuccess
+import elements.Error
+import org.koin.core.KoinApplication
+import org.koin.core.module.Module
+import org.koin.dsl.koinApplication
+import org.koin.dsl.module
+
+/**
+ * The entry point for JVM apps to connect with the backend service
+ * and retrieve [ChatkitCore] instance for further interaction with the SDK.
+ */
+class ChatkitCoreConnector {
+
+    /**
+     * Creates instance of this class for your Chatkit instance
```

Comment:
`Creates an instance of ChatkitCore for your Chatkit instance` - this is hard for a human to parse. We use instance to mean two things in the same sentence.  How about using occurrence for the first instance e.g. `Creates an occurrence of the ChatkitCore class with your Chatkit instance information`

Reply:
"occurrence" is not a technical term here and would be very confusing, I have changed it to "object" though: d3be67e16e0dfb0aa5a067818f363686ed95f435